modules/javafx.controls/src/main/java/javafx/scene/control/ListCell.java

```diff
@@ -28,6 +28,7 @@
 import java.lang.ref.WeakReference;
 import java.util.List;
 
+import javafx.application.Platform;
```

Comment:
Since we changing the file, shouldn't we bump the copyright year too?

Reply:
Thanks, Alex,
We do update the copyright year in batch for all needed files before release. So, it can be skipped in individual fixes.